apis/dataplane/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -88,6 +89,7 @@ type OpenStackDataPlaneServiceSpec struct {
 	// CertsFrom - Service name used to obtain TLSCert and CACerts data. If both
 	// CertsFrom and either TLSCert or CACerts is set, then those fields take
 	// precedence.
+	// DEPRECATED: Will be removed in a future release. Use EDPMServiceType instead.
 	// +kubebuilder:validation:Optional
 	CertsFrom string `json:"certsFrom,omitempty" yaml:"certsFrom,omitempty"`
```

Comment:
We could have dropped this optional field (if there is no update impact) as we've done in multiple other cases till now as there would be no issues installing the new CRD. The crd schema checker which does not allow this can be bypassed by disabling `NoFieldsRemoval`.